ScnSoft-Education/LeetCodeTraining/MergeKSortedLists/MergeKSortedListsSolution.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace LeetCodeTraining.MergeKSortedLists
+{
+    //https://leetcode.com/problems/merge-k-sorted-lists/
+    public static class MergeKSortedListsSolution
+    {
+        public static ListNode MergeKLists(ListNode[] lists)
+        {
+            if (lists.Length == 0)
+            {
+                return null;
+            }
+
+            PriorityQueue<ListNode, int> queue = new PriorityQueue<ListNode, int>();
+            ListNode node = new ListNode();
+            ListNode head = node;
+
+            for (int i = 0; i < lists.Length; i++)
+            {
+                if (lists[i] != null)
+                {
+                    queue.Enqueue(lists[i], lists[i].val);
+                }
+            }
+
+            while (queue.TryDequeue(out ListNode currentNode, out int value))
```

Comment:
Please remove unnecessary variable assignment, because it is never used.
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0059